repl/src/main/scala/com/cloudera/livy/repl/Session.scala

```diff
@@ -168,4 +195,21 @@ class Session(interpreter: Interpreter, stateChangedCallback: SessionState => Un
 
     compact(render(resultInJson))
   }
+
+  private def setJobGroup(statementId: Int): String = {
+    val cmd = Kind(interpreter.kind) match {
+      case Spark() | PySpark() | PySpark3() =>
+        s"""sc.setJobGroup("$statementId", "Job group for statement $statementId")"""
```

Comment:
Scala 2.11's repl  doesn't bind results to result variables like `res0` `res1` hence we don't need to use ignored variable.